packages/terra-date-input/src/DateInputField.jsx

```diff
@@ -221,6 +228,10 @@ const DateInputField = (props) => {
     </legend>
   );
 
+  const format = (DateInputUtil.computedDisplayFormat(displayFormat, intl.locale) === 'month-day-year')
+    ? `(${intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`
+    : `(${intl.formatMessage({ id: 'Terra.date.input.dayFormatLabel' })} ${intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${intl.formatMessage({ id: 'Terra.date.input.yearFormatLabel' })})`;
+
```

Comment:
can we move this logic to datinpututil itself ? as we are using the same in logic in dateinput and field ?

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1230/commits/6d0b38a862d2eb4ae38c5ee585177e32fdda6811